projects/numpy.md

```diff
@@ -0,0 +1,222 @@
+# Numpy 
+___
+
+* [Numpy nima](#numpy-nima)
+* [Arraylar bilan ishlash](#arraylar-bilan-ishlash)
+* [Arraylarning bir qancha funksiyalari](#arraylarning-bir-qancha-funksiyalari)
+* [Tayanch tushunchalar](#tayanch-tushunchalar)
+
+```Console
+-------
+|
+|    |      |
+|  --|--   
+|    |      |
+|
+-------
+```
+## Numpy nima
+___
+
+* NumPy (==Numerical Python==) - bu python kutubxonasidan biri bo'lib, u arraylar (matritsa) bilan ishlashga yordam beradi.
+```python
+import numpy
+
+arr = numpy.array([1, 2, 3, 4, 5])
+
+print(arr)
+```
+* Dastlabki qadamda biz NumPy o'rnatib olamiz.
+```pyhton
+pip install numpy
```

Comment:
`conda install numpy` ga o'zgartishimiz kerak.

Reply:
to'g'irlandi